src/directives/bind.js

```diff
@@ -66,8 +66,10 @@ export function handleAttributeBindingDirective(component, el, attrName, express
         // If an attribute's bound value is null, undefined or false, remove the attribute
         if ([null, undefined, false].includes(value)) {
             el.removeAttribute(attrName)
-        } else {
-            isBooleanAttr(attrName) ? el.setAttribute(attrName, attrName) : el.setAttribute(attrName, value)
+        } else if (isBooleanAttr(attrName)) {
+            el.setAttribute(attrName, attrName)
+        } else if(el.getAttribute(attrName) !== value){
```

Comment:
I think this should be a loose comparison.

All HTML attribute values will be returned as strings, even if they're integers, floats etc, but if you're using an integer as the `value`, this check would fail and never get updated.

Reply:
I think Ryan is right. (It will always be updated since 1 !== '1' but the concept is correct).
Also, why do we want to exclude booleans? I believe this fix should apply to both boolean and non-boolean attributes. Thanks

Reply:
Yup, always pass but fail 🤦‍♂️